packages/common/src/ui-common/form/form.ts

```diff
@@ -882,10 +939,10 @@ export class SubForm<
         return this.form.getSubForm(name);
     }
 
-    updateValue<
-        SK extends Extract<keyof S, string>,
-        S2 extends S[SK] & FormValues
```

Comment:
I thought this `& FormValues` was needed to give Typescript a hint that subforms have values that implement the FormValues interface, but maybe not. Frankly subforms aren't fully fleshed out outside of tests, so it's probably fine.